app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.spec.tsx

```diff
@@ -176,6 +202,68 @@ describe('InPersonLocationStep', () => {
       );
       await findAllByText('in_person_proofing.body.location.location_button');
     });
+
+    it('displays correct pluralization for a single location result', async () => {
+      const { findByLabelText, findByText } = render(
+        <SWRConfig value={{ provider: () => new Map() }}>
+          <InPersonContext.Provider value={{ arcgisSearchEnabled: true }}>
+            <InPersonLocationPostOfficeSearchStep {...DEFAULT_PROPS} />
+          </InPersonContext.Provider>
+        </SWRConfig>,
+      );
+      await userEvent.type(
+        await findByLabelText('in_person_proofing.body.location.po_search.address_search_label'),
+        '800 main',
+      );
+      await userEvent.click(
+        await findByText('in_person_proofing.body.location.po_search.search_button'),
+      );
+      await userEvent.click(
+        await findByText('in_person_proofing.body.location.po_search.search_button'),
+      );
+
+      const searchResultAlert = findByText(
+        t('in_person_proofing.body.location.po_search.results_description', {
+          address: '222 Merchandise Mart Plaza',
+          count: DEFAULT_RESPONSE.length,
+        }),
+      );
+      expect(searchResultAlert).to.exist();
+    });
+
+    before(() => {
+      server.use(
+        rest.post(ADDRESS_SEARCH_URL, (_req, res, ctx) => res(ctx.json(MULTI_LOCATION_RESPONSE))),
+        rest.post(LOCATIONS_URL, (_req, res, ctx) => res(ctx.json([{ name: 'Baltimore' }]))),
+      );
+    });
+
+    it('displays correct pluralization for multiple location results', async () => {
+      const { findByLabelText, findByText } = render(
+        <SWRConfig value={{ provider: () => new Map() }}>
+          <InPersonContext.Provider value={{ arcgisSearchEnabled: true }}>
+            <InPersonLocationPostOfficeSearchStep {...DEFAULT_PROPS} />
+          </InPersonContext.Provider>
+        </SWRConfig>,
+      );
+      await userEvent.type(
+        await findByLabelText('in_person_proofing.body.location.po_search.address_search_label'),
+        '800 main',
+      );
+      await userEvent.click(
+        await findByText('in_person_proofing.body.location.po_search.search_button'),
+      );
+      await userEvent.click(
+        await findByText('in_person_proofing.body.location.po_search.search_button'),
+      );
+      const searchResultAlert = findByText(
+        t('in_person_proofing.body.location.po_search.results_description', {
+          address: '222 Merchandise Mart Plaza',
+          count: MULTI_LOCATION_RESPONSE.length,
+        }),
+      );
+      expect(searchResultAlert).to.exist();
```

Comment:
`findByText` returns a promise, which will be truthy, but I don't think that's what you're actually wanting to test.

```ts
!!new Promise(() => {})
// true
```

I expect you'd want to `await` the `findByText`, which will throw if it never finds the text.

Reply:
> 

Hey Matt, I'm trying to replicate as closely as possible what's happening in `in-person-locations.tsx`:
```javascript
<h3 role="status">
        {!isPilot &&
          t('in_person_proofing.body.location.po_search.results_description', {
            address,
            count: locations?.length,
          })}
 </h3>
```

So I think `t` does take in a second parameter that is an object.

Reply:
> 

Great catch! I'm incorporating that change.

Reply:
Ok gotcha. Yeah you're right. It's not actually translating the content: 

<img width="833" alt="Screen Shot 2023-01-30 at 4 49 38 PM" src="https://user-images.githubusercontent.com/14644234/215603138-64848b19-62b8-4432-ade2-b28519ce3dca.png">

Reply:
> Or it could even append which one it rendered: `results_description.one` or `results_description.other`.

yeah that matches my expectations, we'd render the `one` or the `other` "string" which in this case happens to be the key

Reply:
By default, there is no locale data loaded in the test environment, and the default behavior of the `i18n` library absent locale strings is to [render the given string](https://github.com/18F/identity-idp/blob/34dea94ed3fbde1918d535812f714b73747873ab/app/javascript/packages/i18n/index.ts#L36). This is usually okay for testing, but unfortunately it doesn't account for pluralization, so you either have to be content in testing the parent key, or set up the locale data in the tests with pluralization data ([plain JS approach](https://github.com/18F/identity-idp/blob/main/app/javascript/packages/countdown/countdown-element.spec.ts#L15-L19), [React context approach](https://github.com/18F/identity-idp/blob/34dea94ed3fbde1918d535812f714b73747873ab/spec/javascripts/packages/document-capture/components/review-issues-step-spec.jsx#L76-L87)).